torch/_refs/__init__.py

```diff
@@ -4301,8 +4301,11 @@ def arange(
     type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
 )
 def lerp(start: Tensor, end: Tensor, weight: Union[Tensor, NumberType]):
+    inputs = [start, end]
```

Comment:
in eager, all the inputs participate in computing output strides, not just start and end